### PR TITLE
Use enable_deck=True in docs

### PR DESCRIPTION
**Pull request by thomasjpfan**
## Tracking issue
Related to https://github.com/flyteorg/flyte/issues/5618

## Why are the changes needed?

`disable_deck` is deprecated.

## What changes were proposed in this pull request?

Fixes docs to use `enable_deck=True`.

### CODE REVIEW

**Review by pingsutw — DISMISSED** [review]

**Review by eapolinario — DISMISSED** [review]

File: docs/flyte_fundamentals/visualizing_task_input_and_output.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ how to generate an HTML report from some Python object.
...
-To enable Flyte decks, simply set `disable_deck=False` in the `@task` decorator:
+To enable Flyte decks, simply set `enable_deck=True` in the `@task` decorator:
 
 ```{code-cell} ipython3
 import pandas as pd
 from flytekit import task, workflow
 
 
-@task(disable_deck=False)
+@task(enable_deck=False)
```
Comment (neverett) [inline docs/flyte_fundamentals/visualizing_task_input_and_output.md:32]:
```suggestion
@task(enable_deck=True)
```

File: docs/flyte_fundamentals/visualizing_task_input_and_output.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ from typing import Optional
...
```
Comment (neverett) [inline docs/flyte_fundamentals/visualizing_task_input_and_output.md:54]:
```suggestion
@task(enable_deck=True)
```

File: docs/flyte_fundamentals/visualizing_task_input_and_output.md [hunk 3]
```diff
@@ -168,7 +168,7 @@ function. In the following example, we extend the `iris_data` task with:
 import flytekit
 from flytekitplugins.deck.renderer import MarkdownRenderer, BoxRenderer
 
-@task(disable_deck=False)
+@task(enable_deck=False)
```
Comment (neverett) [inline docs/flyte_fundamentals/visualizing_task_input_and_output.md:171]:
```suggestion
@task(enable_deck=True)
```

File: docs/flyte_fundamentals/visualizing_task_input_and_output.md [hunk 4]
```diff
@@ -220,7 +220,7 @@ except ImportError:
     from typing_extensions import Annotated
 
 
-@task(disable_deck=False)
+@task(enable_deck=False)
```
Comment (neverett) [inline docs/flyte_fundamentals/visualizing_task_input_and_output.md:223]:
```suggestion
@task(enable_deck=True)
```

**Review by neverett — CHANGES_REQUESTED** [review]

**Comment by neverett** [comment]
I think a few of those should be `enable_deck=True`, but let me know if not!

File: docs/flyte_fundamentals/visualizing_task_input_and_output.md (hunk 2 above)
Comment (pingsutw) [reply docs/flyte_fundamentals/visualizing_task_input_and_output.md:54]: good catch, thanks!

**Review by eapolinario — APPROVED** [review]

**Review by neverett — APPROVED** [review]

**Pull request merged**